common/persistence/sql/domains.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE IF NOT EXISTS domains(
+  id VARCHAR(64) UNIQUE NOT NULL,
+  retention_days INT NOT NULL,
+  emit_metric TINYINT(1) NOT NULL,
+/* end domain_config */
+  config_version BIGINT NOT NULL,
+  db_version BIGINT,
+/* domain */
+  name VARCHAR(255) PRIMARY KEY NOT NULL,
+  status INT NOT NULL,
+  description VARCHAR(255) NOT NULL,
+  owner_email VARCHAR(255) NOT NULL,
+/* end domain */
```

Comment:
NIT: can you move all the stuffs in domain section to the top? This is probably what people want to see first.